internal/session/rewrapping_test.go

```diff
@@ -45,38 +46,38 @@ func TestRewrap_sessionCredentialRewrapFn(t *testing.T) {
 	}
 
 	kmsWrapper, err := kmsCache.GetWrapper(context.Background(), prj.PublicId, kms.KeyPurposeDatabase)
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
I updated all of these to `require` as we don't want to test to continue on any of these errors (since it may invalidate assumptions about values used in the rest of the test, leading to confusing panics).